### DISCUSSION TITLE
x-bind: not working on custom blade component

### DISCUSSION THREAD

**Discussion by maweitzman** [alpinejs/alpine#2500] [1. Help]
I'm trying to wire up `x-bind:` on one of my custom blade components like so:

```html
<x-core.svgs.base
    size="7"
    color="brand"
    x-bind:name="isFollowing ? 'unfollow' : 'follow'"
/>
```

However, because `x-bind` is not set as a prop on my `Base` component, it will not accept it. I know I can edit the component to accept, or use, any passed in Alpine properties, but is there any other way to accomplish this?

**Comment by zepfietje** [comment]
Can you show your component definition?
Do you echo passed attributes in your component using `{{ $attributes }}`?
